backup.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/python3
+import os, logging, subprocess, ast, smtplib
+from time import strftime, localtime
+from socket import gaierror
+
+BACKUP_NAME=os.environ.get("BACKUP_NAME")
+BACKUP_ENCRYPTION_KEY=os.environ.get("BACKUP_ENCRYPTION_KEY")
+BACKUP_SCHEDULE=os.environ.get("BACKUP_SCHEDULE")
+BACKUP_LOCATION=os.environ.get("BACKUP_LOCATION")
+BACKUP_PRUNE=os.environ.get("BACKUP_PRUNE")
+BACKUP_NOW=os.environ.get("BACKUP_NOW")
+BACKUP_VERBOSE=os.environ.get("BACKUP_VERBOSE")
+BORG_CUSTOM_ARGS=ast.literal_eval(os.environ.get("BORG_CUSTOM_ARGS"))
```

Comment:
Missed this!  I fixed to be optional (as I intended).  It looks like the literal_eval function doesn't support defaults, so I moved the evaluation to later in the script, only if the var is set.  BACKUP_PRUNE starts a whole new subprocess call, the BORG_CUSTOM_ARGS value gets injected into the regular backup command to avoid having to write the same block of code twice with only one variable change.